### ISSUE TITLE
the Electron Security Warning is normal or is it a bug?

### ISSUE THREAD

**Issue by DenisLevy76** [electron-react-boilerplate/electron-react-boilerplate#2881]
## Summary

I'm receiving this warning in the terminal and I don't know if it would be a bug in the template
![image](https://user-images.githubusercontent.com/62116847/119051095-1c3fa500-b999-11eb-9f0c-6db3437914b9.png)

<!---

**Comment by amilajack** [comment]
That is fixed in #2875

**Comment by Aromokeye** [comment]
The error still persist

**Comment by svirid132** [comment]
I was helped by writing the http-equiv meta tag inside the head tag of the index.ejs file:  
```
<head>
    <meta charset="utf-8" />
    <meta http-equiv="Content-Security-Policy" content="script-src 'self' 'unsafe-inline'">
    <title>Hello Electron React!</title>
</head>
```

**Comment by amilajack** [comment]
Fixed in b41a813d3475fe9697cbc11ac173bdb613a9925a
